setuptools/config.py

```diff
@@ -245,33 +245,40 @@ def _parse_file(cls, value):
         directory with setup.py.
 
         Examples:
-            include: LICENSE
-            include: src/file.txt
+            file: LICENSE
+            file: README.rst, CHANGELOG.md, src/file.txt
 
         :param str value:
         :rtype: str
         """
+        include_directive = 'file:'
+
         if not isinstance(value, string_types):
             return value
 
-        include_directive = 'file:'
         if not value.startswith(include_directive):
             return value
 
-        current_directory = os.getcwd()
-
-        filepath = value.replace(include_directive, '').strip()
-        filepath = os.path.abspath(filepath)
+        spec = value[len(include_directive):]
+        filepaths = (os.path.abspath(path.strip()) for path in spec.split(','))
+        return '\n'.join(
+            self._read_local_file(path)
+            for path in filepath
+            if os.path.isfile(path)
+        )
 
-        if not filepath.startswith(current_directory):
+    @staticmethod
+    def _read_local_file(filepath):
+        """
+        Read contents of filepath. Raise error if the file
+        isn't in the current directory.
+        """
+        if not filepath.startswith(os.getcwd()):
```

Comment:
`os.getcwd()` generates a syscall on each loop iteration. It would be preferable to call it once in `__init__` and then reuse saved value.

Reply:
Sounds like a premature optimization to me. The added complication of the logic doesn't justify the rare, few syscalls.

Reply:
I saw dudes on the Internet complaining that setuptools is slow, that's why this came to my mind.